frontend/src/components/common/PartySelectField.js

```diff
@@ -0,0 +1,187 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { debounce } from "lodash";
+import { compareTwoStrings } from "string-similarity";
+import PropTypes from "prop-types";
+import CustomPropTypes from "@/customPropTypes";
+
+import { Icon, Divider, AutoComplete } from "antd";
+import { Field } from "redux-form";
+import RenderLargeSelect from "./RenderLargeSelect";
+
+import { fetchParties, setAddPartyFormState } from "@/actionCreators/partiesActionCreator";
+import { getRawParties, getLastCreatedParty } from "@/selectors/partiesSelectors";
+import { createItemMap, createItemIdsArray } from "@/utils/helpers";
+
+// Set organization to only see organizations or set people to only see people. Displays both people and organizations by default.
```

Comment:
I was intending to refer to the props below. Would it make more sense to put those inline with those two props?

Reply:
Either inline or on the line right above the relevant props, instead of above the entire props block, would make more sense to me, yes. I thought this might have been an outdated comment

Reply:
I'll move them 👍

Reply:
Moved